### PR TITLE
Add enumerations for pushchair

### PR DESCRIPTION
**Pull request by syversenkr**
Add the following facility enumerations :

AccommodationFacilityEnumeration
•	pushchair
•	wheelchair

MobilityFacilityEnumeration
•	suitableForPushchairs

LuggageCarriageEnumeration
•	pushchairsAllowed

### CODE REVIEW

**Comment by syversenkr** [comment]
This matter was discussed in the BaseCamp thread https://3.basecamp.com/3256016/buckets/2570434/messages/3829763349

**Comment by skinkie** [comment]
@Aurige we can, and open an issue concerning the documentation which should be updated.

**Comment by syversenkr** [comment]
Noted that @nick-knowles mentioned this being added to the "next" branch on Basecamp. But as this branch does not yet seem to be established (neither this repo or Nicks repo), I decided to push here as the fix will also be pushed to the Entur master branch - and correlating NeTEx Java Model (Entur) - when sync'ed with prior updates here in the CEN repo.

But if you prefer this PR to a new branch, that of course fine with us.

**Comment by skinkie** [comment]
@syversenkr I'll take care of it.

File: xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd [hunk 1]
```diff
@@ -245,6 +248,8 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="panoramaCoach"/>
 			<xsd:enumeration value="pullmanCoach"/>
 			<xsd:enumeration value="standing"/>
+			<xsd:enumeration value="wheelchair"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:251]: Can you move wheelchair below pushchair, like suitableForPushchairs, suitableForWheelchairs  etc.

**Review by skinkie — CHANGES_REQUESTED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd:251]: also agree on the better ordering

Comment (syversenkr) [reply xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd]: Check. And check.

**Review by skinkie — APPROVED** [review]

**Pull request merged**